built-in-policies/policyDefinitions/Kubernetes/ContainerNoPrivilegeEscalation_EnforceOPAConstraint.json

```diff
@@ -0,0 +1,54 @@
+{
+  "properties": {
```

Comment:
Built-in policies are only checked into this repo by an automated system each time we release updates to Azure.  They live in the AzureUX-ARM repo internally.  Contact me (cheggert) if you need a pointer to their location.

Reply:
Sweet, I've removed this file, will coordinate with Ramya for committing the policy definitions